kind/common.sh

```diff
@@ -210,9 +211,15 @@ function _kind_up() {
 }
 
 function main() {
-    _kind_up
-
-    echo "cluster '$CLUSTER_NAME' is ready"
+    case $1 in
+    up)
+        _kind_up
+        echo "cluster '$CLUSTER_NAME' is ready"
+        ;;
+    down)
```

Comment:
To make the code elegant, I would suggest to name the function in same style: down => _kind_down

Reply:
nice suggestion will add this change as well
Thanks🙂